### DISCUSSION TITLE
An Experiment into "VEDA" Token and Smart Contract on Bitcoin with Ordinals

### DISCUSSION THREAD

**Discussion by UXVpcyBjdXN0b2RpZXQgaXBzb3MgY3VzdG9kZXM** [VedaOrg/Veda-docs#8] [General]
**I.**
Greetings GitHub community,

I hope this message finds you in good spirits. I am reaching out to engage in a discussion and seek support for the Veda Protocol—an initiative that holds promise for the evolution of bitcoin. I am honored to share that, as part of this endeavor, the deployment of the inaugural token under the Veda Protocol has been deployed by me, and it goes by the name "VEDA."

**II.**
The backing of the community is pivotal for the success of the Veda Protocol. Your feedback, valuable suggestions, and insights are eagerly sought. I extend an invitation to developers and enthusiasts within the GitHub community to delve into the deployment of the "VEDA" token, and to contribute thoughts on its potential applications.

**III.**
Your perusal of the provided information and your thoughts on the proposed concepts and implementations are welcomed. I hold the belief that the convergence of Bitcoin Ordinals and the Veda Protocol opens avenues for exciting possibilities in the realm of bitcoin innovation.

ID: b9a547145960ffdc7087a63a4b7604e7d3072996d375eae24ae396e1e12f079di0

CA: 0xb37ebd46f53d9c05f57c9f4943c249ecd78b0cff

Best regards,

**UXVpcyBjdXN0b2RpZXQgaXBzb3MgY3VzdG9kZXM**

**Comment by FarmerJoeCodes** [comment]
Love this initiative and would support it!

**Comment by DrBitstein** [comment]
This is awesome and I can't wait to play with it!

**Comment by 0xNakazutra** [comment]
Is it possible to open up a community group?

**Comment by PsyGogitit** [comment]
UXVpcyBjdXN0b2RpZXQgaXBzb3MgY3VzdG9kZXM  

Quis custodiet ipsos custodes?
